test/test_const_folding.py

```diff
@@ -14,7 +14,7 @@ def _check_ast_count(desired_count:int, t:Tensor):
 
 class TestUnaryOpsConstFolding(unittest.TestCase):
```

Comment:
These tests should be updated properly, instead of just bumping the kernel counts. Positive counts means they are not folded. I think it's either keep as 0 and fixed the test, or keep as 0 and add expectedfailure (for the failed ones only and keep folded ones tested)